### ISSUE TITLE
making .bib file available in the log folder

### ISSUE THREAD

**Issue by hstojic** [poldracklab/fmriprep#1444]
Hello,

for those of us using LaTeX to write papers it would be great to include the .bib file used to generate the CITATION files. It's a bit cumbersome to find it in a singularity container, so that I can simply copy those references in the bib file for my article. For the reference, if anybody else is interested in how to dig it up in your singularity container, here is the command:

```bash
singularity exec fmriprep-X.X.X.img cat /usr/local/miniconda/lib/python3.6/site-packages/fmriprep/data/boilerplate.bib
```

Thanks!

**Comment by effigies** [comment]
Yes, this would be good to include in the derivatives. Would you be interested in contributing a patch?

**Comment by hstojic** [comment]
Sure, just point me in the direction of a script that generates CITATION files and I'll send a pull request.

**Comment by effigies** [comment]
@oesteban The CITATIONs were your baby... Do you know off the top of your head where it's copied?

**Comment by oesteban** [comment]
CITATION versions are generated here: https://github.com/poldracklab/fmriprep/blob/db956bb9ec43256ad9e31904f9e438ea6170fac8/fmriprep/cli/run.py#L749

FYI, the bib references are appended into the html report, within the Latex tab.

**Comment by effigies** [comment]
I think if we're going to output the `.tex` file, we might as well output the `.bib` file, though. So we should probably just add a `shutil.copyfile()` there.

**Comment by hstojic** [comment]
Ah, that's true, they are in .html report of every subject, I haven't looked there actually. Still, it would save some time looking for them if the whole bib file is simply placed in the logs folder as well.

**Comment by hstojic** [comment]
I filed a quick pull-request: #1446 

I have to admit, didn't do the whole patch thing as described in the fmriprep docs, it seemed small enough change to go without it.

**Comment by effigies** [comment]
Closed in #1446.
